apps/public-docsite/package.json

```diff
@@ -38,6 +38,8 @@
     "@fluentui/react-examples": "^8.34.4",
     "@fluentui/react-experiments": "^8.4.11",
     "@fluentui/react-file-type-icons": "^8.5.7",
+    "@fluentui/react-icons": "^2.0.154-beta.5",
+    "@fluentui/react-make-styles": "^9.0.0-beta.4",
```

Comment:
In case anyone wonders what the pushed and reverted commit was about--I considered pinning react-make-styles to a slightly older version to prevent unwanted yarn behavior once react-make-styles is updated (I was concerned it would install a different version and cause a lock file change), but I experimented with it locally and there wasn't an issue. So I guess yarn is better about deduplicating compatible semver ranges when they refer to a package within the monorepo.